apps/src/gamelab/AnimationPicker/animationPickerModule.js

```diff
@@ -140,7 +140,7 @@ function loadImageMetadata(sourceUrl, callback) {
       sourceSize: {x: image.width, y: image.height},
       frameSize: {x: image.width, y: image.height},
       frameCount: 1,
-      frameRate: 15
+      frameDelay: 4
```

Comment:
I don't think these are equivalent; at 30fps a frameDelay of 4 represents an animation frameRate of 7.5fps.  Intentional?